components/brave_new_tab_ui/components/default/braveToday/customize/CustomizePage.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import styled from 'styled-components'
+import { getLocale } from '../../../../../common/locale'
+import Button from '../../../../../web-components/button'
+import Flex from '../../../Flex'
+import { useBraveNews } from './Context'
+import { BackArrow } from './Icons'
+
+const BackButtonContainer = styled.div`
+  all: unset;
+  flex: 1;
+
+  &> button {
+    --inner-border-size: 0;
+    --outer-border-size: 0;
+    padding: 0;
+
+    &:hover {
+      --inner-border-size: 0;
+      --outer-border-size: 0;
+    }
+  }
+`
+
+const Header = styled.span`
+  font-weight: 500;
+  font-size: 16px;
+  color: var(--text01);
+  flex: 5;
+  text-align: center;
+`
+
+const Spacer = styled.div`flex: 1;`
```

Comment:
This is an interesting way to center the text. It breaks down if the word "Back" is extremely long, but that's very unlikely!